.github/workflows/notebooks.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       run: |
         if [ "${{ github.event_name }}" == "pull_request" ]; then
           # Only check notebooks modified in this pull request
-          readarray -t changed_notebooks < <(git diff --name-only main | grep '\.ipynb$' || true)
+          readarray -t changed_notebooks < <(git diff --name-only main --diff-filter="d" | grep '\.ipynb$' || true)
```

Comment:
So, exclude deleted files from the notebook check? Sounds good to me.

However, it looks like the syntax for diff-filter does not include quotes (""). Should be:

--diff-filter=d

reference: https://git-scm.com/docs/git-diff#Documentation/git-diff.txt-code--diff-filterACDMRTUXBcode